packages/charts/src/chart_types/xy_chart/state/selectors/get_cursor_pointer.ts

```diff
@@ -34,6 +36,7 @@ export const getPointerCursorSelector = createCustomCachedSelector(
     projectedValues,
     { chartDimensions },
     isBrushAvailable,
+    tooltipState,
```

Comment:
This context is not obvious I think we rename this to be clear.

```suggestion
    annotationTooltipState,
```

Reply:
Smart suggestion cb44d3e